src/o365/teams/commands/report/teams-report-deviceusagedistributionusercounts.ts

```diff
@@ -39,18 +48,62 @@ class TeamsReportDeviceUsageDistributionUserCountsCommand extends GraphCommand {
     request
       .get(requestOptions)
       .then((res: any): void => {
-        cmd.log(res);
+        let content: string = '';
+
+        if (args.options.output && args.options.output.toLowerCase() === 'json') {
+          let reportdata = this.getJsonReport(res);
+          content = JSON.stringify(reportdata);
+
+          if (!args.options.outputFile) {
+            cmd.log(reportdata);
+          }
+        }
+        else {
+          content = res;
+
+          if (!args.options.outputFile) {
+            cmd.log(content);
+          }
+        }
```

Comment:
In the spec you mentioned `csv` as an additional report format but I don't see any implementation of it here. Have you decided to skip it for now? If so, it would be good to update the spec of the related issue for consistency.

Reply:
CSV and Text formats are same logic so its not written specific to CSV

Reply:
Removed the CSV format from the commands.